### PR TITLE
Feat : Added loginPage

### PR DESCRIPTION
**Pull request by RahulSingh9131**
this branch contains
1. login page (without authentication)

### CODE REVIEW

File: src/pages/LoginPage.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import {Link,useNavigate} from "react-router-dom";
+
+function LoginPage() {
+
+    const [email,setEmail]=useState("");
+    const [password,setPassword]=useState("");
+    const navigate=useNavigate();
+
+    const fetchUser=()=>{
+      if(email && password){
```
Comment (raman1999) [inline src/pages/LoginPage.js:11]: make use of api call later, if email and password are in backend users.js

**Review by raman1999 — APPROVED** [review]
looks good, just basic functionality added

File: src/pages/LoginPage.js (hunk 1 above)
Comment (RahulSingh9131) [reply src/pages/LoginPage.js:11]: yes sure

**Review by shubhamkumar648 — APPROVED** [review]
Good use of variables in css. reaming code looks fine

**Pull request merged**